### PR TITLE
Added generic "property filtering" to `/missions` and `/runtimes`

### PR DESCRIPTION
**Pull request by quintesse**

### CODE REVIEW

File: core/core-impl/src/test/java/io/fabric8/launcher/core/impl/catalog/ScriptingRhoarBoosterPredicateTest.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
 package io.fabric8.launcher.core.impl.catalog;
```
Comment (gastaldi) [inline core/core-impl/src/test/java/io/fabric8/launcher/core/impl/catalog/ScriptingRhoarBoosterPredicateTest.java]: This class needs to be moved to BCS

**Comment by gastaldi** [comment]
It would be nice to have a test in BoosterCatalogEndpointIT with this new feature as well

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @quintesse for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

File: core/core-impl/src/test/java/io/fabric8/launcher/core/impl/catalog/ScriptingRhoarBoosterPredicateTest.java (hunk 1 above)
Comment (quintesse) [reply core/core-impl/src/test/java/io/fabric8/launcher/core/impl/catalog/ScriptingRhoarBoosterPredicateTest.java]: Done

**Pull request merged**